input/pagecontent/iti-71.md

```diff
@@ -569,8 +570,15 @@ There are no CapabilityStatement resources defined for this transaction.
 
 ### Security Consideration
 
-IUA Authorization Clients, IUA Authorization Servers and IUA Resource Server actors SHALL use the JWS (signed) alternative
-of the JWT token as specified in the IUA Trial Implementation. The JWE alternative SHALL not be used.
+IUA Authorization Clients, IUA Authorization Servers and IUA Resource Server actors SHALL support the JWS (signed) 
+alternative of the JWT token as specified in the IUA Trial Implementation. To ensure the authenticity and integrity, 
+the IUA Authorization Server SHALL sign the JWT token with its private key and IUA Resource Servers SHALL verify 
+the signature of the JWT token with the Authorization Server's public key. The JWE alternative SHALL not be used.
+
+To ensure the authenticity and integrity, IUA Authorization Clients SHALL sign requests to the token endpoint of 
+the IUA Authorization Server with the clients' private key as defined in RFC 9421 `HTTP Message Signatures`. 
+The signature SHALL cover the entire request content. The IUA Authorization Server SHALL verify the requests' 
+signature with the clients' public key exchanged during the client registration process. 
```

Comment:
That's quite light on details, given the number of possibilities offered by the RFC. In particular:

1. There are ~14 different algorithms defined, we can't expect Authorization Servers to implement them all. We should require support for one or two algorithms, and allow other algorithms if needed.
2. We should require a maximum validity duration for the signature, forcing verifiers to verify it. The `expiration` field would be optional for signers, and the field `created` would become required.
3. We should specify requirements on the signature parameters. Otherwise, lazy signers could omit everything but `content-digest`.